flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1228,7 +1228,7 @@ var A320_Neo_UpperECAM;
                     {
                         message: "STROBE LT OFF",
                         isActive: () => (
-                            this.isInFlightPhase(6, 7, 8) && !this.getCachedSimVar("LIGHT STROBE ON", "Bool")
+                            this.isInFlightPhase(6, 7, 8) && !this.getCachedSimVar("LIGHT STROBE OFF", "Bool")
```

Comment:
It only prevents the message from appearing at all, when departing with strobe light in off position, instead of changing the strobe behavior itself.